metadata.yml

```diff
@@ -0,0 +1,64 @@
+id: pga
+name: Platform Governance Archive
+tagline: Major global social media services
+description: |
+  The **Plateform Governance Archive** (PGA) collection tracks the terms of major global social media services.
+  
+  These data are maintained and analysed by the [Platform Governance Archive](https://www.platformgovernancearchive.org/) at the Universität Bremen's [Center for Media Communication and Information Research (ZeMKI)](https://www.uni-bremen.de/zemki).
+  
+  This initiative offers researchers, journalists and citizens the tools to analyze how platforms structure and regulate communication and interaction in our societies.
+  
+  It also aims to promote greater transparency and accountability of these powerful digital services.
+dataset: https://github.com/OpenTermsArchive/pga-versions/releases
+declarations: https://github.com/OpenTermsArchive/pga-declarations
+versions: https://github.com/OpenTermsArchive/pga-versions
+snapshots: https://github.com/OpenTermsArchive/pga-snapshots
+donations: https://opencollective.com/opentermsarchive
```

Comment:
We have inconsistency here. In the doc it's `donation` and it is not documented in the API

Reply:
Discussed in synchro, we're sticking with `donations` and making the necessary changes in the API.